docs/website/docs/dlt-ecosystem/verified-sources/filesystem.md

```diff
@@ -63,6 +67,57 @@ To get GCS/GDrive access:
 For more info, see how to
 [create service account](https://support.google.com/a/answer/7378726?hl=en).
 
+To get GCS/GDrive access using **OAuth** credentials:
+
+You need to create a GCP account to get OAuth credentials if you don't have one. To create one,
+follow these steps:
+
+1. Initialize the verified source as explained in ["initialize-the-verified-source"](#initialize-the-verified-source) to set up OAuth credentials.
```

Comment:
I understand that for markdown it is just fine to have `1.`s and nevertheless should it enumerate this as usual 1,2,3...?